src/test/import-utils_test.ts

```diff
@@ -62,132 +65,138 @@ suite('import-utils', () => {
+        <script>Polymer({is: "my-element"})</script>
+      `;
+
+      const expected = `
+        <html><head><link rel="import" href="polymer/polymer.html">
+        <link rel="stylesheet" href="my-element/my-element.css">
+        </head><body><dom-module id="my-element" assetpath="my-element/">
+        <template>
+        <style>:host { background-image: url("my-element/background.svg"); }</style>
+        <div style="position: absolute;"></div>
+        </template>
+        </dom-module>
+        <script>Polymer({is: "my-element"})</script></body></html>
+      `;
 
       const ast = parse5.parse(html);
       importUtils.rewriteImportedUrls(ast, importDocPath, mainDocPath);
 
       const actual = parse5.serialize(ast);
-      assert.equal(actual, expected, 'relative');
+      assert.equal(stripSpace(actual), stripSpace(expected), 'relative');
     });
 
-    test.skip('Resolve Paths with <base>', () => {
-      const htmlBase = [
-        '<base href="zork">',
-        '<link rel="import" href="../polymer/polymer.html">',
-        '<link rel="stylesheet" href="my-element.css">',
-        '<dom-module id="my-element">',
-        '<template>',
-        '<style>:host { background-image: url(background.svg); }</style>',
-        '</template>',
-        '</dom-module>',
-        '<script>Polymer({is: "my-element"})</script>'
-      ].join('\n');
-
-      const expectedBase = [
-        '<html><head>',
-        '<link rel="import" href="my-element/polymer/polymer.html">',
-        '<link rel="stylesheet" href="my-element/zork/my-element.css">',
-        '</head><body><dom-module id="my-element" assetpath="my-element/zork/">',
-        '<template>',
-        '<style>:host { background-image: url("my-element/zork/background.svg"); }</style>',
-        '</template>',
-        '</dom-module>',
-        '<script>Polymer({is: "my-element"})</script></body></html>'
-      ].join('\n');
+    test('Resolve Paths with <base>', () => {
+      const htmlBase = `
+        <base href="zork">
+        <link rel="import" href="../polymer/polymer.html">
+        <link rel="stylesheet" href="my-element.css">
+        <dom-module id="my-element">
+        <template>
+        <style>:host { background-image: url(background.svg); }</style>
+        </template>
+        </dom-module>
+        <script>Polymer({is: "my-element"})</script>
+      `;
+
+      const expectedBase = `
+        <html><head>
+        <link rel="import" href="polymer/polymer.html">
+        <link rel="stylesheet" href="my-element/my-element.css">
+        </head><body><dom-module id="my-element" assetpath="my-element/">
+        <template>
+        <style>:host { background-image: url("my-element/background.svg"); }</style>
+        </template>
+        </dom-module>
+        <script>Polymer({is: "my-element"})</script></body></html>
+      `;
 
       const ast = parse5.parse(htmlBase);
-      // pathRewriter.acid(ast, inputPath);
+      importUtils.debaseDocument(importDocPath, ast);
```

Comment:
`Document <base> tag emulation` is its own suite now with focus only on the `rewriteDocumentToEmulateBaseTag` function effects now.